src/features/vars-schema/hooks/vars-update-trigger.ts

```diff
@@ -316,24 +316,25 @@ addEnvHooks('device_service_environment_variable', async (args) => {
 	];
 });
 
+// Normally we don't expect these to return any match, since our clients
+// * POST image env vars before the release & images gets marked as successful
+// * never PATCH them after that point
+// * DELETEs are expected to normally only happen if the release gets DELETED,
+//   in which case the should_be_running__release will change and trigger an update of its own.
 addEnvHooks('image_environment_variable', async (args) => {
 	if (args.req.body.release_image != null) {
 		return {
-			image_install: {
+			should_be_running__release: {
 				$any: {
-					$alias: 'ii',
+					$alias: 'r',
 					$expr: {
-						installs__image: {
+						status: 'success',
+						release_image: {
```

Comment:
Can you use `contains_image` to make this match the state get endpoint, and add a comment to the top of this file that the having the queries match the path the state get endpoint uses to get env vars for a device is the way to go. Essentially this captures if a subset of the state-get query will change so the correct thing is to follow the same path (and also means it'll have the same performance characteristics, which we take real care to optimize for in the state-get)